armory/adapt/pytorch.py

```diff
@@ -0,0 +1,1126 @@
+"""
+Adaptive Attack Framework
+
+Design goals:
+    1) Easy to understand
+    2) Easy to modify
+    3) Easy to compose
+    4) Fast and efficient
+    5) Minimal dependencies (framework-specific)
+
+User stories:
+    a) Initialize an attack with an interesting input
```

Comment:
For the user story `Initialize an attack with an interesting input`, not sure I'm following exactly what's meant since you can pass whatever to ART's `generate()` or in the case here,`__call__()`. Or do you mean the init within the epsilon ball that gets added to the input? If so, it looks like this code restricts you to either random or None.

Reply:
Yes, the desire is to enable the init within the epsilon ball. The code needs to be modified to enable passing in an init function instead of those two choices.